content/docs/README.md

```diff
@@ -0,0 +1,122 @@
+# Docs
+
+Welcome to Neon docs! This folder contains the source code of the [Neon docs](https://neon.tech/docs/).
+
+## Basic information
+
+1. Every single Markdown file in this folder will be turned into a docs page.
+2. Folder and file names should follow kebab-case.
+3. `slug` is generated based on the folder structure and file names inside this folder.
+
+   In order to see page `slug`, you can [start the project](../../README.md#run-the-website), [open GraphQL Playground](https://www.gatsbyjs.com/docs/using-graphql-playground/#accessing-the-playground) and [run query](https://www.gatsbyjs.com/docs/how-to/querying-data/running-queries-with-graphiql/#example-of-using-graphiql) that will display all docs pages with fields `title` and `slug`:
+
+   ```graphql
+   {
+     allMdx(filter: { fileAbsolutePath: { regex: "/content/docs/" } }) {
+       nodes {
+         frontmatter {
+           title
+         }
+         slug
+       }
+     }
+   }
+   ```
+
+   <details>
+    <summary>Screenshot</summary>
+
+    <img width="1536" alt="image" src="https://user-images.githubusercontent.com/20713191/173828228-6ba0f953-77ef-4ca4-8290-613ed6db7cac.png">
+   </details>
+
+4. Page `path` is generated by combining [`DOCS_BASE_PATH`](../../src/constants/docs.js) and page `slug`.
+5. There is no need to add `h1` to the page since it will be displayed automatically with the value from `title` field.
+
+## Fields
+
+Right now Markdown files accept 2 fields:
+
+1. `title` — title of the page (required)
+2. `isDraft` — flag that says the page is not ready yet. It won't appear in production but will appear in the development mode.
+
+> ⚠️ Please note that project won't build if one of Markdown files doesn't have even a single required field!
```

Comment:
⚠️ Please note that the project won't build if at least one of the Markdown files is missing a required field.